applications/TopologyOptimizationApplication/custom_utilities/structure_response_function_utilities.h

```diff
@@ -0,0 +1,307 @@
+// ==============================================================================
+//  KratosTopologyOptimizationApplication
+//
+//  License:         BSD License
+//                   license: TopologyOptimizationApplication/license.txt
+//
+//  Main authors:    Baumgärtner Daniel, https://github.com/dbaumgaertner
+//                   Octaviano Malfavón Farías
+//                   Eric Gonzales
+//					 Philipp Hofer
+//					 Erich Wehrle
+//
+// ==============================================================================
+
+#if !defined(KRATOS_STRUCTURE_RESPONSE_FUNCTION_UTILITIES_H_INCLUDED)
+#define  KRATOS_STRUCTURE_RESPONSE_FUNCTION_UTILITIES_H_INCLUDED
+
+// System includes
+#include <iostream>
+#include <string>
+#include <algorithm>
+
+// External includes
+#include <pybind11/pybind11.h>
```

Comment:
I will not repeat myself, including copy and pastes a whole file in the source code...